static/_redirects

```diff
@@ -579,3 +579,4 @@
 /platform/concepts/projects /docs/platform/about#project
 /platform/concepts/environments /docs/platform/about#environment
 /platform/concepts /docs/platform/about
+/pulse/concepts /docs/pulse/what-is-pulse
```

Comment:
can't highlight more lines. Could these static redirects be moved to above line 534? The bottom section is supposed to be for dynamic redirects only (things with `*`)

Reply:
done, thanks for catching this!